app/models/measure.rb

```diff
@@ -245,6 +239,34 @@ def with_certificate_types_and_codes(certificate_types_and_codes)
       where(combined_conditions)
     end
 
+    def join_footnotes
+      association_right_join(:footnotes)
+        .exclude(measures__measure_sid: nil)
+    end
+
+    def with_footnote_type_id(footnote_type_id)
+      return self if footnote_type_id.blank?
+
+      where(footnotes__footnote_type_id: footnote_type_id)
+    end
+
+    def with_footnote_id(footnote_id)
+      return self if footnote_id.blank?
+
+      where(footnotes__footnote_id: footnote_id)
+    end
+
+    def with_footnote_types_and_ids(footnote_types_and_ids)
+      return self if footnote_types_and_ids.none?
+
+      conditions = footnote_types_and_ids.map do |type, id|
+        Sequel.expr(footnotes__footnote_type_id: type) & Sequel.expr(footnotes__footnote_id: id)
```

Comment:
Is not `&` a bitwise and?
I guess it should be `&&` or `and`

Reply:
This is a logical operator, not a bitwise operator in Sequel, basically


![image](https://github.com/trade-tariff/trade-tariff-backend/assets/8156884/a0116c5d-e75a-47e8-b84d-ebd52bc4f53e)